### PR TITLE
feat(demo): new documentation page `Testing`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [X] Docs update

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [X] No

### CODE REVIEW

File: projects/demo/src/modules/info/testing/testing.template.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                        target="_blank"
+                        href="https://github.com/TinkoffCreditSystems/argus"
+                        >in its repo's README</a
+                    >
+                    or use default ones.
+                </li>
+            </ol>
+        </li>
+    </ol>
+</tui-doc-page>
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/info/testing/testing.template.html:137]: Maybe make separate subpages for token and argument?
And I think we can add argus to friendly libraries page

**Comment by nsbarsukov** [comment]
@vladimirpotekhin
I made new changes. Look into them, please.

File: projects/demo/src/modules/info/testing/testing.template.html (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/info/testing/testing.template.html]: done

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by hakimio** [comment]
@nsbarsukov `TinkoffCreditSystems/argus` leads to Github 404 page. Is it a private repository?

**Comment by waterplea** [comment]
Yes, sorry, it is currently being audited before it's made public. Hopeful it goes live this week.

**Pull request merged**